### ISSUE TITLE
Portable VirtualBox does not run if VirtualBox is installed directly.

### ISSUE THREAD

**Issue by DavidBerdik** [vboxme/Portable-VirtualBox#81]
I am interested in using Portable VirtualBox to run VirtualBox 5 on a system that has VirtualBox 6 installed as the main version. It seems however that if VirtualBox is already installed on the system, that installation is used in place of the portable version, although the the portable VMs are used. Is there a way to work around this or is this a deliberate design choice?

**Comment by runarbu** [comment]
Yes, this is a deliberate design choice. If it was not like this we would break the installed VirtualBox unfortunately.

As you mention the  portable VMs are used, so it was my impression that it would work fine. How is this casing issues for you?

**Comment by p060477** [comment]
hi runar!
is there now finally a possibility of running VB portable with bridged network with win 10...??
..so strange that this issue has not been solved since years and years...
what you think...??
:(

**Comment by DavidBerdik** [comment]
@runarbu It causes an issue for me because I am using Portable VirtualBox with Antidetect. Antidetect is a fork of VirtualBox that is designed to evade virtual machine detection, so I therefore cannot use the standard VirtualBox installation on my machine. I have worked around this for now by temporarily uninstalling VirtualBox, using Portable VirtualBox to run Antidetect, and reinstalling VirtualBox when I am done.

I suppose the issue with breaking the host VirtualBox installation comes from the fact that you'd need to remove the drivers for it. Would it be possible to temporarily remove the drivers and put them back when done?
